plasmapy/particles/particle_class.py

```diff
@@ -421,9 +421,38 @@ class Particle(AbstractPhysicalParticle):
     ``'transition metal'``, ``'uncharged'``, and ``'unstable'``.
     """
 
-    @staticmethod
-    def _validate_arguments(argument, Z, mass_numb):
+    def __init__(
+        self,
+        argument: ParticleLike,
+        mass_numb: Integral = None,
+        Z: Integral = None,
+    ):
+
+        # If argument is a Particle instance, then construct a new
+        # Particle instance for the same particle.
+
+        if isinstance(argument, Particle):
+            argument = argument.symbol
+
+        self._inputs = argument, mass_numb, Z
+
+        self._initialize_attributes_and_categories()
+        self._store_particle_identity()
+        self._assign_particle_attributes()
+        self._add_charge_information()
+        self._add_half_life_information()
```

Comment:
A question that came up during live code review was about whether or not these methods should take arguments.  My motivation for not having arguments comes mostly from pages 40–41 of in Chapter 3 of _Clean Code_.  Also, while I was working on this, I found that the `Particle.__init__` was harder to refactor when there were arguments to these methods.  A worry about this approach is that not having arguments could sweep some of the story under the rug.  We essentially decided to try out this approach for the time being.  If this ends up leading to confusion later on, then we can change this later on.